src/components/admin/MainIssueController.jsx

```diff
@@ -156,8 +156,26 @@ export default class MainIssueController extends FalcorController {
               return false;
             }
           }
+          if (/<a.*?href\s*?=\s*?["'](?!http).*?>/.test(article.html)) {
+            if (!window.confirm(
+              `${article.title} is in non-absolute format,
```

Comment:
Also, it's not the title that is in non-absolute format, it's a url in it, so that should be made clear, also, it'd actually be even better if we told them exactly what the url is, which you can do with capturing parenthesis and using the `.match` method. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match and for more generic documentation https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions.

You can just rerun that if the if statement passes and get the capture group, you should only need to modify the regex by adding a `(.*?)["']` clause after the `(?!http)` one.

Reply:
Should I delete the {article.title} part and replace it with the .match method? Or do I include both to make it clear that it refers to that specific article and url?